src/main/java/kostka/moviecatalog/dto/UserDto.java

```diff
@@ -0,0 +1,54 @@
+package kostka.moviecatalog.dto;
+
+import javax.validation.constraints.NotEmpty;
+
+public class UserDto {
```

Comment:
Just a thought: Such DTO classes might be final as well.

Reply:
I can do it, yes

Reply:
But optional. Maybe a matter of taste.